templates/scripts.tx

```diff
@@ -18,3 +18,5 @@
 	<script src="/static/js/highlight.pack.js" type="text/javascript"></script>
 	<script>hljs.initHighlightingOnLoad();</script>
 : }
+
+<script src="//jag.duckduckgo.com:5000/livereload.js"></script>
```

Comment:
It's a different port number so I need to specify a hostname. @jbarrett is there a variable that I can use to get the hostname?